doc/user-guide/client-certificate.md

```diff
@@ -44,7 +44,7 @@ For the details please refer to [XEP-0178 Best Practices for Use of SASL EXTERNA
 You need to enable one of the following authentication backends by using the [`auth_method` option](../Advanced-configuration.md#authentication) in the MongooseIM configuration file.
```

Comment:
```suggestion
You need to enable one of the following authentication methods by using the [`auth_method` option](../Advanced-configuration.md#authentication) in the MongooseIM configuration file.
```

Reply:
Also, please change `ingore` to `ignore` in line 169, I couldn't comment there.

Reply:
I did nothing to this file yet, but I will fix this.